lib/tasks/generateCachebusterInfo.js

```diff
@@ -10,17 +37,20 @@ const resourceFactory = require("@ui5/fs").resourceFactory;
  * @param {module:@ui5/fs.AbstractReader} parameters.dependencies Reader or Collection to read dependency files
  * @param {Object} parameters.options Options
  * @param {string} parameters.options.namespace Namespace of the application
+ * @param {string} parameters.options.signatureType Type of signature to be used (time|hash)
```

Comment:
signatureType is optional and defaults to type, right? So, this needs to be marked here with the brackets in the JSDoc [parameters.options.signatureType] and should be explained like ", defaults to time"

Reply:
Sorry, my fault, I'm not really used to JSDoc!

I saw there's a way to properly define enumerable and default values too, something like:
```
@param {('bar'|'baz')} [foo='bar'] An optional param foo with a default value 'bar'
```
I'll fix the code accordingly.